### PR TITLE
Improve archive command

### PR DESCRIPTION
**Pull request by wasdennnoch**
- add error message when no valid number is provided
- add archived message count to output file name and message
- remove unnecessary fields from output
- add message attachments to output
- use default values so all fields always exist in output

**What does this PR do:**
- [ ] This PR changes internal functions, modules and/or utilities
  - [ ] This PR modifies the Extension API
- [x] This PR modifies commands
  - [x] This PR changes command functions
  - [ ] This PR changes metadata for the command, updated in `commands.js`
  - [ ] This PR removes and/or adds commands
- [ ] This PR modifies Web processing and/or content
  - [ ] This PR modifies static/ejs content
  - [ ] This PR modifies request processing (endpoint functions)
  - [ ] This PR modifies paths to existing resources
- [ ] This PR **only** includes non-code changes, like changes to default configurations, README, typos, etc.

### CODE REVIEW

File: Commands/Public/archive.js [hunk 1]
```diff
@@ -24,39 +31,49 @@ module.exports = async ({ Constants: { Colors } }, documents, msg, commandData)
...
+				name: a.name,
+				url: a.attachment,
+			})),
+		}));
+		const fileBuffer = Buffer.from(JSON.stringify(messages, null, 2));
 		try {
 			await msg.channel.send({
-				content: `Here you go! ✅`,
-				files: [new Attachment(fileBuffer, `archive-${msg.guild.name}-${msg.channel.name}-${Date.now()}.json`)],
+				content: `Here you have the last ${count} messages! ✅`,
```
Comment (vladfrangu) [inline Commands/Public/archive.js]: Should be messages.size instead. Safer!

File: Commands/Public/archive.js [hunk 2]
```diff
@@ -24,39 +31,49 @@ module.exports = async ({ Constants: { Colors } }, documents, msg, commandData)
...
+				files: [new Attachment(fileBuffer, `archive-${msg.guild.name}-${msg.channel.name}-${count}-${Date.now()}.json`)],
```
Comment (vladfrangu) [inline Commands/Public/archive.js]: I kinda disagree with it being in the file name, but I'd agree with it being in the actual JSON. Something like:
```json
{
  "totalMessages": messages.size,
  "messages": messages.map()
}
```

**Review by vladfrangu — CHANGES_REQUESTED** [review]
I love you too! Have some feedback!

File: Commands/Public/archive.js (hunk 2 above)
Comment (wasdennnoch) [reply Commands/Public/archive.js]: While we're at it, I could also add other metadata in the file such as archive creation time, server name etc.

Comment (vladfrangu) [reply Commands/Public/archive.js]: Sure

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**